src/classes/Batchs/MinAccountOppBatch.cls

```diff
@@ -0,0 +1,64 @@
+/**
+ * Created by Mariya on 17.01.2023.
+ */
+
+public with sharing class MinAccountOppBatch implements Database.Batchable<sObject>, Schedulable, Database.Stateful {
+
+    public Iterable<sObject> start(Database.BatchableContext param1) {
+        return Database.getQueryLocator('SELECT Id FROM Account');
```

Comment:
There could be another more reliable solution. Database.Stateful is not used in your code, then why is it implemented?

You can query children records (Opportunities) and then use finish() method along with Database.Stateful.
   `private final Map<Id, List<Decimal>> opportunitiesAmountsByAccountsIds;
    private Logger logger;
    private Decimal sumOfMinimumAmounts = 0;

    public SumMinOpportunitiesAmountsBatch() {
        this.opportunitiesAmountsByAccountsIds = new Map<Id, List<Decimal>>();
        this.logger = new Logger();
    }`

Use constructor to init logger just once and map as well, so it is not reset between chunks.
And you have to UPSERT, not insert metadata.
_finish()_
`upsert new OrgConfiguration__c(
                Name = 'AccsMinOppsAmountsSummary',
                LesserAmounts__c = this.sumOfMinimumAmounts
        );

        logger.createInfoLog(Summary.BATCH_JOB, SumMinOpportunitiesAmountsBatch.class,
                'Custom setting Org Configuration updated to ' + this.sumOfMinimumAmounts);

        logger.flush();`